packages/core/src/salt-core.css

```diff
@@ -0,0 +1,42 @@
+@import "accordion/Accordion.css";
```

Comment:
This is not automated, right? Seems like we could easily forget to add files here if the runtime injection is the default

Reply:
You can forget to do it yes....but there is a command in package.json that will recreate the entry file for you. Will see if we can change the order of the build commands